Sources/Receipt/ListReceiptViewPresenter.swift

```diff
@@ -130,12 +130,12 @@ final class ListReceiptViewPresenter {
             }
     }
 
-    private func groupReceiptsByMonth(_ receipts: [HyperwalletReceipt]) {
-        let groupedSections = Dictionary(grouping: receipts,
+    private func groupReceiptsByMonth(_ receipts: [HyperwalletReceipt]?) {
+        let groupedSections = Dictionary(grouping: receipts ?? [],
```

Comment:
do not change this method. It should group only if there are receipts. There's no point og calling this method if there's no receipts to be grouped.

Reply:
Leave it like ``` private func groupReceiptsByMonth(_ receipts: [HyperwalletReceipt]) ```